tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": "./src",
```

Comment:
This should be `./`, though I think it would be good to move all of the files under an `src` folder in a separate PR

Reply:
Good catch!